pkgs/development/tools/jsonnet-bundler/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "jsonnet-bundler";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "jsonnet-bundler";
+    repo = "jsonnet-bundler";
+    rev = "v${version}";
+    sha256 = "0pk6nf8r0wy7lnsnzyjd3vgq4b2kb3zl0xxn01ahpaqgmwpzajlk";
+  };
+
+  vendorSha256 = "1h1d32zi1pfb7spprsacxg8ygrigswqqngrc8gsdzn1xbldphl5g";
+
+  doCheck = false;
```

Comment:
Can you add a comment why checks are disabled?

Reply:
Ah thanks for the reminder, tests are working and are enabled now.